spring-integration-core/src/test/java/org/springframework/integration/channel/config/DispatchingChannelParserTests.java

```diff
@@ -37,18 +37,20 @@
 import org.springframework.messaging.Message;
 import org.springframework.messaging.MessageChannel;
 import org.springframework.messaging.MessageHandler;
-import org.springframework.test.context.ContextConfiguration;
-import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import org.springframework.test.context.junit.jupiter.SpringJUnitConfig;
 
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+import static org.mockito.Mockito.mock;
 
 /**
  * @author Mark Fisher
  * @author Oleg Zhurakousky
+ * @author Artem Bilan
+ *
  * @since 1.0.3
  */
-@RunWith(SpringJUnit4ClassRunner.class)
-@ContextConfiguration
+@SpringJUnitConfig
 public class DispatchingChannelParserTests {
```

Comment:
One general comment: since you are modifying a test, why don't we clean this up as JUnit 5 style tests? i.e - keeping the test class and the test methods with default access?

Reply:
This is not enforced style, so I leave it up personal preferences. Since this is an existing class, I wouldn’t bother removing those `public` words. 
The new test is just a copy/paste artifact , so still no care 😅